src/OrchardCore.Modules/OrchardCore.Users/Filters/TwoFactorAuthenticationAuthorizationFilter.cs

```diff
@@ -36,7 +36,7 @@ public async Task OnAuthorizationAsync(AuthorizationFilterContext context)
     {
         ArgumentNullException.ThrowIfNull(context);
 
-        if (context.HttpContext?.User?.Identity?.IsAuthenticated == false ||
+        if (context.HttpContext?.User.Identity.IsAuthenticated == false ||
```

Comment:
If we just strictly follow the nullable annotations of .NET, `Identity` can be null here as well.